lib/kitchen/provisioner/policyfile_zero.rb

```diff
@@ -96,9 +96,10 @@ def create_sandbox
       # (see Base#run_command)
       def run_command
         level = config[:log_level] == :info ? :auto : config[:log_level]
-        chef_client_bin = sudo(config[:chef_client_path])
+        
+        cmd = "#{sudo(config[:chef_client_path])} --local-mode".
+          tap { |str| str.insert(0, "& ") if powershell_shell? }
```

Comment:
I have a feeling this is going to be easier to read if you do something like this:

```
prefix = "& " if powershell_shell?
cmd = "#{prefix}#{sudo(config[:chef_client_path])} --local-mode"
```

Other than that, standard question: what test can be reasonably done here?